source/get-started/index.rst

```diff
@@ -46,4 +46,13 @@ Install in a virtual machine
    :glob:
 
    virtual-machine-install/*
-   ../../guides/maintenance/increase-virtual-disk-size.rst
\ No newline at end of file
+   ../../guides/maintenance/increase-virtual-disk-size.rst
+
+Install in the cloud
```

Comment:
I would recommend the phrase "Deploy" over "Install" when referring to cloud images. Same note regarding the navigation/path.